sql/migrations/20201010150010_world.sql

```diff
@@ -0,0 +1,123 @@
+UPDATE `creature_template` SET `spell_list_id` = 79370 WHERE `entry` = 7937;
+UPDATE `creature_template` SET `ai_name` = 'EventAI' WHERE `entry` = 7937;
+
+
+-- Arch Druid Fandral Staghelm 3516
+REPLACE INTO `creature_spells` (`entry`, `name`, `spellId_1`, `probability_1`, `castTarget_1`, `targetParam1_1`, `targetParam2_1`, `castFlags_1`, `delayInitialMin_1`, `delayInitialMax_1`, `delayRepeatMin_1`, `delayRepeatMax_1`, `scriptId_1`, `spellId_2`, `probability_2`, `castTarget_2`, `targetParam1_2`, `targetParam2_2`, `castFlags_2`, `delayInitialMin_2`, `delayInitialMax_2`, `delayRepeatMin_2`, `delayRepeatMax_2`, `scriptId_2`, `spellId_3`, `probability_3`, `castTarget_3`, `targetParam1_3`, `targetParam2_3`, `castFlags_3`, `delayInitialMin_3`, `delayInitialMax_3`, `delayRepeatMin_3`, `delayRepeatMax_3`, `scriptId_3`, `spellId_4`, `probability_4`, `castTarget_4`, `targetParam1_4`, `targetParam2_4`, `castFlags_4`, `delayInitialMin_4`, `delayInitialMax_4`, `delayRepeatMin_4`, `delayRepeatMax_4`, `scriptId_4`, `spellId_5`, `probability_5`, `castTarget_5`, `targetParam1_5`, `targetParam2_5`, `castFlags_5`, `delayInitialMin_5`, `delayInitialMax_5`, `delayRepeatMin_5`, `delayRepeatMax_5`, `scriptId_5`, `spellId_6`, `probability_6`, `castTarget_6`, `targetParam1_6`, `targetParam2_6`, `castFlags_6`, `delayInitialMin_6`, `delayInitialMax_6`, `delayRepeatMin_6`, `delayRepeatMax_6`, `scriptId_6`, `spellId_7`, `probability_7`, `castTarget_7`, `targetParam1_7`, `targetParam2_7`, `castFlags_7`, `delayInitialMin_7`, `delayInitialMax_7`, `delayRepeatMin_7`, `delayRepeatMax_7`, `scriptId_7`, `spellId_8`, `probability_8`, `castTarget_8`, `targetParam1_8`, `targetParam2_8`, `castFlags_8`, `delayInitialMin_8`, `delayInitialMax_8`, `delayRepeatMin_8`, `delayRepeatMax_8`, `scriptId_8`) VALUES (35160, 'Darnassus - Arch Druid Fandral Staghelm', 20698, 100, 1, 0, 0, 0, 0, 0, 2, 3, 0, 20702, 100, 0, 0, 0, 0, 3, 5, 25, 35, 0, 20701, 100, 0, 0, 0, 0, 8, 12, 26, 30, 0, 20699, 100, 1, 0, 0, 0, 5, 7, 17, 21, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0);

```

Comment:
So after comparing with the existing spell list, what you changed is you removed the main ranged spell flag from Wrath, meaning he will now do the melee attacking state animation while casting, and will not stop chasing his target, so he will be making small steps between each cast. Is that really how it works in classic? Normally ranged mobs stand still and just spam their main spell when at range. Also you removed Hurricane from the list. And you changed Rejuvenation to self only, without checking health. Does he really cast it on himself on a timer, even when on full health?

Reply:
I cannot find any evidence that he uses Hurricane.